site/public/js/models/ioobjects/other/IC.js

```diff
@@ -64,7 +64,7 @@ class IC extends IOObject {
         renderer.rect(0, 0, size.x, size.y, this.getCol(), '#000000', 1);
 
         for (var i = 0; i < this.inputs.length; i++) {
-            var name = this.inputObjects[i].getName();
+            var name = String(i);
```

Comment:
Why is this change here? I believe the IC has intended behavior of taking the names of the input objects